index.js

```diff
@@ -32,6 +32,13 @@ Cypress.Commands.add('percySnapshot', (name, options) => {
   name = name || cy.state('runnable').fullTitle();
 
   cy.then(async () => {
+    if (Cypress.config('isInteractive')) {
```

Comment:
We should also check for a bypass config option. Something so if somebody really needs to run Percy in interactive mode (such as for debugging) it can be done without modifying the source inside node_modules.
```suggestion
    if (Cypress.config('isInteractive') && 
        !Cypress.config('enablePercyInteractiveMode')) {
```